proposals/0000-add-support-for-css-color-function.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Add support for CSS color() function
```

Comment:
Do you want to make three different proposals (JS, iOS and Android) or do you want to do a single one?
I'd probably would like to have a single proposal with the whole picture.

In that case, it is better to name the RFC something like: `[RFC] Add Support for Wide Gamut (DisplayP3) Colors to React Native`

Reply:
We definitely want a single proposal. Thanks for the suggestion! 😄